README.md

```diff
@@ -197,7 +218,7 @@ connection.release(preparedStatement: preparedStatement) { result in
 ### Table creation
 Swift-Kuery enables you to create tables on the database server.
 
```

Comment:
Change line 244 to remove the ", do" from the end of the sentence "To create the table in the database, do:"